Started by user [8mha:////4CDaLzao4A67utYR77ls98WC8cTIZkXIASznBGiVkvolAAAAmR+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAz2EgZe/dLi1CL9rNS87My8YgDGtiJjwgAAAA==[0mJenkins
Running as SYSTEM
Building in workspace /var/jenkins_home/workspace/ansible-users-db
[8mha:////4EDIeoeEAtfSqs9BRpJQXfYbi4unNtESNKsaxkQWjp8YAAAAqx+LCAAAAAAAAP9b85aBtbiIQSujNKU4P0+vIKc0PTOvWC8xrzgzOT8nv0gvMbkkEyjhCKb88ktSdwVsu2efuqidiYHRh4EDIu2ZUsIg5JOVWJaon5OYl64fXFKUmZduXVHEIAU1ODk/rzg/J1XPGUKDDGKAAEYmBoaKghIGFRMTEyPz1FRz3TQTMzNdk9TUFF1L8+QkXfO0NAtLIzODRAMLAwAZXf9NrwAAAA==[0m[ansible-users-db] $ ansible-playbook /var/jenkins_home/ansible/people.yml -i /var/jenkins_home/ansible/hosts -f 5 -e PEOPLE_AGE=21
/usr/local/lib/python3.5/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography.exceptions import InvalidSignature

PLAY [web1] ********************************************************************

TASK [Gathering Facts] *********************************************************
[0;32mok: [web1][0m

TASK [Tranfer template to web server] ******************************************
[0;33mchanged: [web1][0m

PLAY RECAP *********************************************************************
[0;33mweb1[0m                       : [0;32mok=2   [0m [0;33mchanged=1   [0m unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   

Finished: SUCCESS
